RELEASE.md

```diff
@@ -41,6 +41,8 @@
 | `AbstractDataset`          | `AbstractDataSet`          | `kedro.io.core` |
 | `AbstractVersionedDataset` | `AbstractVersionedDataSet` | `kedro.io.core` |
 
+* `layer` attribute at the top level is deprecated and will be removed in Kedro 0.19.0. Please move `layer` inside the `metadata` -> `kedro-viz` attributes.
```

Comment:
```suggestion
* Using the `layer` attribute at the top level is deprecated; it will be removed in Kedro version 0.19.0. Please move `layer` inside the `metadata` -> `kedro-viz` attributes.
```